config/README.rst

```diff
@@ -268,6 +268,9 @@ updated. Example:
     additional-install = [
         "sudo apt-get update && sudo apt-get install -y libxml2-dev libxslt-dev",
         ]
+    test-commands = [
+        "tox -f ${{ matrix.config[1] }}",
```

Comment:
typo: `-e` instead of `-f`

Reply:
`-f` is intended here, it is an option of https://pypi.org/project/tox-factor/, maybe I should add as an example in the option above.